src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -42,10 +48,9 @@ export interface AutocompleteProps {
 // generated *.d.ts files.
 
 export const Autocomplete: React.FunctionComponent<AutocompleteProps> & {
-  ComboBox: typeof ComboBox;
-  TextField: typeof TextField;
+  ComboBox: typeof ComboBoxOld;
+  TextField: typeof ComboBox.TextField;
```

Comment:
Both text fields extend the base Polaris textfield without adding additional API's